source/CommonJobs/CommonJobs.Application.MyMenu/GetEmployeeMenuCommand.cs

```diff
@@ -47,13 +47,22 @@ public override EmployeeMenuDTO ExecuteWithResult()
 
             var menuDefinition = ExecuteCommand(new GetMenuDefinitionCommand(employeeMenu.MenuId));
 
-            var lastOrder = RavenSession.Load<MenuOrder>(MenuOrder.GenerateId(employeeMenu.MenuId, menuDefinition.LastOrderDate));
-            if (lastOrder != null)
-            {
-                lastOrder.IsOrdered = true;
-            }
+            var today = DateTime.Now.Date;
+            var tomorrow = today.AddDays(1);
+            var todayOrderId = MenuOrder.GenerateId(employeeMenu.MenuId, today);
+
+            var todayOrder = RavenSession.Load<MenuOrder>(todayOrderId);
+
+            // IS THIS NECESSARY? The orders are created with IsOrdered = true
+            //if (todayOrder != null)
+            //{
+            //    todayOrder.IsOrdered = true;
+            //}
```

Comment:
It seems to be a message for me, maybe this code was there in order to support old documents without `IsOrdered` property. If current data and code confirms that it is not necessary, please remove it.